src/GameLogic/MiniGames/ChaosCastleContext.cs

```diff
@@ -89,7 +89,7 @@ public ChaosCastleContext(MiniGameMapKey key, MiniGameDefinition definition, IGa
     protected override TimeSpan RemainingTime => this._remainingTime;
 
     /// <inheritdoc />
-    protected override int MinimumPlayerCount => 2;
+    protected override int MinimumPlayerCount => 1;
```

Comment:
I accept this change for now, but I'd like to add this as setting to the MiniGameDefinition, so everyone can decide how it should be on his server. The normal behavior on the official server is, that it requires two players to start the event.